core/src/main/resources/hudson/slaves/JNLPLauncher/help_pt_BR.properties

```diff
@@ -21,13 +21,3 @@
 # THE SOFTWARE.
 
 blurb=Permite que um agente seja conectado ao Jenkins controller sempre que estiver pronto.<br>\
-Em um modo o <a href="https://en.wikipedia.org/wiki/Java_Web_Start" rel="noopener noreferrer" target="_blank">Java Web Start</a> \u00E9 usado. \
```

Comment:
is this right? it seems to be just cutting off the text. there's a \ without any continuation

Reply:
You're right... I checked against the original file and looks like translation_tool.pl messed up in this one. Compare with the original file in English:

```
blurb=\
  Allows an agent to be connected to the Jenkins controller whenever it is ready.<br>\
  In one mode, <a href="https://en.wikipedia.org/wiki/Java_Web_Start" rel="noopener noreferrer" target="_blank">Java Web Start</a> is used. \
  In this case, a JNLP file must be opened on the agent machine, \
  which will establish a TCP connection to the Jenkins controller. \
  (Other launch methods use a JNLP file but not Java Web Start, or do not use a JNLP file at all.)<br>\
  This means that the agent need not be reachable from the controller; \
  the agent just needs to be able to reach the controller. \
  If you have enabled security via the <i>Configure Global Security</i> page, \
  you can customize the port on which the Jenkins controller will listen for incoming agent connections.<br>\
  By default, the agent will launch a GUI, but it is also possible to run \
  an agent without a GUI, for example as a Windows service.
```
Maybe because de sentence starts with a backslash (`\`)?
I'll fix that manually and send again.